crates/rapier_testbed3d/Cargo.toml

```diff
@@ -39,7 +39,7 @@ other-backends = ["physx", "physx-sys", "glam"]
 features = ["parallel", "other-backends"]
 
 [dependencies]
-nalgebra = { version = "0.33", features = ["rand", "glam027"] }
+nalgebra = { version = "0.33", features = ["rand", "glam029"] }
```

Comment:
I'm weary of physx comment saying we depend on glam 0.27 ; I imagine that would need an optional dependency to glam compat ?

Reply:
I didn't detect any unexpected behaviour in this branch compared to master or other previous release :ok_hand: